### PR TITLE
temporary fix for broken faa cucumber steps

### PR DESCRIPTION
**Pull request by RyanEddyIC**
# PR Checklist

Please check if your PR fulfills the following requirements:
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes/features), and they use `let` helpers and `before` blocks.
- [ ] For all UI changes, there is Cucumber coverage.
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, the reasoning is documented in the PR and code.
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run them is documented in both the PR and the code.
- [ ] There are no inline styles added.
- [ ] There is no inline JavaScript added.
- [ ] There is no hard-coded text added/updated in helpers/views/JavaScript. New/updated translation strings do not include markup/styles unless there is supporting documentation.
- [ ] Code does not use `.html_safe`.
- [ ] All images added/updated have alt text.
- [ ] Does not bypass RuboCop rules in any way.

# PR Type
What kind of change does this PR introduce?:

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix, Migration, or Report (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket:

# A brief description of the changes:

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and indicate which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

File: components/financial_assistance/app/views/financial_assistance/applicants/tax_info.html.erb [hunk 1]
```diff
@@ -207,7 +207,7 @@
           <div class="col-xs-12">
             <div class="back-link">
               <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-              <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b><%= l10n("faa.tax_info.back") %></b> </a> <span><%= l10n("faa.tax_info.back_paranthetical") %></span>
+              <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b><%= l10n("faa.tax_info.back") %></b> </a> <span><%= l10n("faa.tax_info.back_paranthetical") %></span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/applicants/tax_info.html.erb:210]: replace `javascript:void(0)` with `#`. Should add       `event.preventDefault()` to the top of the function call if it isn't there already. I don't think the javascript:void will pass the CSP test.

File: components/financial_assistance/app/views/financial_assistance/applications/preferences.html.erb [hunk 2]
```diff
@@ -190,7 +190,7 @@
             <div class="col-xs-12">
               <div class="back-link">
                 <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-                <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+                <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/applications/preferences.html.erb:193]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/financial_assistance/app/views/financial_assistance/applications/submit_your_application.html.erb [hunk 3]
```diff
@@ -263,7 +263,7 @@
             <div class="col-xs-12">
               <div class="back-link">
                 <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-                <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+                <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/applications/submit_your_application.html.erb:266]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/financial_assistance/app/views/financial_assistance/benefits/index.html.erb [hunk 4]
```diff
@@ -571,7 +571,7 @@
         <div class="col-xs-12">
           <div class="back-link">
             <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-            <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+            <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/benefits/index.html.erb:574]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/financial_assistance/app/views/financial_assistance/deductions/index.html.erb [hunk 5]
```diff
@@ -274,7 +274,7 @@
         <div class="col-xs-12">
           <div class="back-link">
             <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-            <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+            <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/deductions/index.html.erb:277]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/financial_assistance/app/views/financial_assistance/incomes/index.html.erb [hunk 6]
```diff
@@ -337,7 +337,7 @@
         <div class="col-xs-12">
           <div class="back-link">
           <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-            <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+            <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/incomes/index.html.erb:340]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/financial_assistance/app/views/financial_assistance/incomes/other.html.erb [hunk 7]
```diff
@@ -529,7 +529,7 @@
       <div class="col-xs-12">
         <div class="back-link">
           <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-          <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+          <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/financial_assistance/app/views/financial_assistance/incomes/other.html.erb:532]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

File: components/ui_helpers/app/views/workflow/step.html.erb [hunk 8]
```diff
@@ -41,7 +41,7 @@
         <div class="col-xs-12">
           <div class="back-link">
             <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-            <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
+            <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b>BACK TO ALL HOUSEHOLD MEMBERS</b> </a> <span>(Go back and work on another person)</span>
```
Comment (bbodine1) [inline components/ui_helpers/app/views/workflow/step.html.erb:44]: replace javascript:void(0) with #. Check for `event.preventDefault()` at the top of the function.

**Review by bbodine1 — APPROVED** [review]

**Review by anncaballeroo — APPROVED** [review]

**Pull request merged**